client/components/info-link/docs/example.jsx

```diff
@@ -0,0 +1,23 @@
+/**
+* External dependencies
+*/
+import React, { Component } from 'react';
+
+/**
+* Internal dependencies
+*/
+import InfoLink from 'components/info-link';
+
+class InfoLinkExample extends Component {
+	static displayName = 'InfoLink';
```

Comment:
Apparently, examples are using `displayName` here to override how it's shown in the react dev tools. So, every example component, is shown with the name of the component they're showcasing.